packages/app-content-pages/src/shared/components/Sidebar/Sidebar.js

```diff
@@ -24,19 +24,12 @@ const StyledButton = styled(Button)`
         background: ${props.theme.global.colors['accent-1']};
       `}
   }
-
-  &:hover,
```

Comment:
I removed the "bold on hover" feature of the sidebar because it's not worth the extra effort of addressing https://github.com/zooniverse/front-end-monorepo/pull/5578#discussion_r1394989855